blackJack.html

```diff
@@ -0,0 +1,32 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>21 Black Jack</title>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script src="./blackJack.js"></script>
+    <link rel="stylesheet" href="./blackJack.css">
+</head>
+<body>
+    <h1>21 Black Jack</h1>
+    <div id="results">
+        <p id="playerResults"></p>
+    </div>        
+    <button id="button2">Start Game  </button>
+    <button id="stay">Stay</button>
+    <button id="button">Hit Card</button>
+   
+    <div id="deckId">
```

Comment:
You do not need to display the id of the current deck on the page. You just need to grab it in your js file.